docs/source/api_reference/geodata.rst

```diff
@@ -0,0 +1,81 @@
+.. raw:: html
+    :file: embed_widgets/geodata.html
+
+GeoData
+=======
+
+``GeoData`` is an ``ipyleaflet`` class that allows you to visualize a ``GeoDataFrame`` on the Map.
+
+GeoDataFrame
```

Comment:
I suppose you can remove this section, it does not add much more information about the layer, apart from the link to the deopandas data structure. Maybe you can put the link on the line above? Smth like:
```
``GeoData`` is an ``ipyleaflet`` class that allows you to visualize a `GeoDataFrame <http://geopandas.org/data_structures.html>_` on the Map.
```